### DISCUSSION TITLE
Welcome to ScratchAddons Discussions!

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#502] [Discussion]
## 👋 Welcome to our new forum!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Note**: We are migrating question and discussion issues here. That means that if you have a question or something to discuss, you should post it [here](https://github.com/ScratchAddons/ScratchAddons/discussions/category_choices)!

**Comment by retronbv** [comment]
I am yoyitsm3m8 i am big coder boi and i help with ideas, and coding

**Reply by TheColaber** [reply]
Hello, I'm @TheColaber I have done some addons for this extension and of course the other things that aren't addons. I'm on the discord server and I'm pretty good at JS. I'm good at not listening to W_L when he tells me not to work on an issue.

**Comment by Hans5958** [comment]
Hello, I'm Hans5958. I maintain the website and contribute some code and ideas on Scratch Addons.

**Comment by jeffalo** [comment]
hi im jeffalo. i write spagetti code and get yelled at by w_l :)

**Reply by Explosion-Scratch** [reply]
XD

**Comment by Explosion-Scratch** [comment]
I am [--Explosion--](https://scratch.mit.edu/users/--explosion--) on scratch and I love to code in JavaScript, HTML and CSS and Python! With ScratchAddons I report bugs, suggest features and I coded the old version of the website (~300 commits)! (The current version is maintained by @Hans5958) Right now I'm working on [my personal site](http://explosion.cf) and [my JavaScript projects](http://js-stuff.cf) and some stuff on my [Repl.it account](https://repl.it/@ExplosionScratc).

**Reply by Hans5958** [reply]
Nah, this is new. We discussed it on the Discord server.

**Reply by Explosion-Scratch** [reply]
Cool!

**Comment by GrahamSH-LLK** [comment]
Hi! I'm @GrahamSH-LLK! I made the See Full Areas addon, and have contributed to many places across ScratchAddons! Right now, I have `73` commits to the main repo!

**Reply by Hans5958** [reply]
> **GrahamSH-LLK** marked then unmarked their comment as the answer 2 minutes ago

lmao

**Reply by GrahamSH-LLK** [reply]
I was seeing if it would let you have multiple answers. Seems kinda lacking that it doesn't.

**Comment by BoomerScratch** [comment]
https://github.com/BoomerScratch/BoomerScratch/blob/master/README.md

**Comment by GrahamSH-LLK** [comment]
This is pretty much done, so I'm going to mark it.

**Reply by Hans5958** [reply]
no lol this is just a simple introductory post

**Comment by jemunm378** [comment]
Nice!

**Comment by Explosion-Scratch** [comment]
lol

**Comment by ghost** [comment]
I am MasimbaTheHuman, I wish I knew CSS

**Reply by daniel4-scratch** [reply]
Just have a crack at it and learn it! Try learn it at https://www.w3schools.com/css/default.asp, It took while for me to get use to css, now I know alot of it off by heart.

**Comment by Illodrus** [comment]
I am oof_le_oof, I just F4F on Scratch and make stupid projects. I need this addon for my workflow though

**Reply by retronbv** [reply]
what addon

**Reply by Illodrus** [reply]
my g. scratchaddons

**Reply by retronbv** [reply]
ScratchAddons isn't an add-on, it's a extension.

**Reply by Hans5958** [reply]
It's an add-on for Firefox users, just like an extension for Chome users.

**Reply by retronbv** [reply]
> It's an add-on for Firefox users, just like an extension for Chome users.

Oh same thing

**Reply by WorldLanguages** [reply]
We try to use the word "extension" even for the Firefox addon to avoid confusion.
